app/assets/javascripts/details/views/observer_card.js

```diff
@@ -16,6 +16,7 @@ ObserverCardController = (function(){
     this.$el.html(html);
     this._selectize();
     this._hideUntilSelect();
+    $('.card-for-activity').trigger('activity-card:update');
```

Comment:
Can you pull this out and put it in the c2.js file? The `observer_card.js` file should only touch things in the `this.el` context.

You could create an event that gets triggered in the `observer_card.js` and listen to it in the `c2.js` file.